src/panels/lovelace/cards/hui-todo-list-card.ts

```diff
@@ -319,10 +334,16 @@ export class HuiTodoListCard extends LitElement implements LovelaceCard {
                     "ui.panel.lovelace.cards.todo-list.no_unchecked_items"
```

Comment:
It's a bit weird to show this if there are todo-items without status. Maybe only show this if both `uncheckedItems` and `itemsWithoutStatus` are empty?